modules/ROOT/pages/websockets/websocket-about-websocket-connector.adoc

```diff
@@ -0,0 +1,60 @@
+= About the Websockets Connector (Design Center)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== Why use Websockets
+
+WebSockets are a key piece of modern integration due to their ability to:
+
+* Seamlessly work on top of the existing HTTP infrastructure.
+* Avoid polling
+* Implement server push patterns
+* Establish bi-directional and full duplex communication between server and client.
+
+== Basic components
+
+There are two basic socket types:
+
+* Inbound: represents a websocket that will open upon receiving an incoming connection request from a client that supports websocket communication.
+* Outbound: represents a websocket that will open a connection to other websockets through the 'connect' operation.
+
+All websockets, both inbound and outbound, will have their own unique socket ID to identify them.
+
+== Basic Operations Overview
+
+Using the WS or WSS protocol, the Websocket connector performs one of the following operations:
+
+* Inbound Listener/On Outbound Socket Message
++ Which will start executing a flow upon receiving an incoming message.
+* Send
++ Send message to a specific socket.
+* Broadcast
++ Send a given message to several sockets classified by path, type and group.
+* Open Outbound Socket
++ Open a connection to an external websocket.
+* On Inbound Connection
++ Start executing a flow when an external connection request is received and connection is successfully established.
+* Close
++ Close a connection established with another websocket.
+* On Socket Closed
++ Start executing a flow when a connection to a socket of any kind (INBOUND/OUTBOUND) has been closed.
+
+Websockets are built upon HTTP configuration and Listeners can be used to trigger flows in the same way as HTTP Listeners can.
+
+They even allow for both an HTTP and a Websocket Listener to be listening on the same port.
+
+While HTTP Connector provides only one Listener and one Requester which fulfill the Server and Client roles respectively, because Websockets allow full-duplex communication, we can establish communication using one Listener/Requester pair of type Outbound or one Listener/Requester pair of type Inbound. All mentioned operations are available for both types of sockets, we will distinguish them between client, server and connection management operations.
```

Comment:
Wording is very confusing and seems to be missing the point. Are you trying to explain why there is one single config instead of being splitted in listener-config and requester-config like http? Is that something that should be explained here or is that a design consideration that only belongs in the spec?